LICENSE

```diff
@@ -1,14 +1,21 @@
-Copyright 2022 T-Systems Multimedia Solutions GmbH
+MIT License
 
-Permission is hereby granted, free of charge, to any person obtaining a copy of this software and associated
-documentation files (the "Software"), to deal in the Software without restriction, including without limitation
-the rights to use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies of the Software,
-and to permit persons to whom the Software is furnished to do so, subject to the following conditions:
+Copyright (c) 2023 Deutsche Telekom MMS GmbH
```

Comment:
Copyright (c) 2022-2023 Deutsche Telekom MMS GmbH

or without a year for easier maintenance